frontend/vue/components/Syllabus/SyllabusView.vue

```diff
@@ -68,7 +68,7 @@ export default defineComponent({
   },
   mounted () {
     getCourseList().then((courses) => {
-      const learningPathCourses = courses.filter(course => course.type === 'learning-path')
+      const learningPathCourses = courses.filter(course => course.type !== 'docs')
```

Comment:
maybe rename variable to `availableCourses` or something along those lines

Reply:
great suggestion @vabarbosa 👏